netlogo-gui/src/main/lab/gui/ProtocolEditable.scala

```diff
@@ -121,6 +122,14 @@ class ProtocolEditable(protocol: LabProtocol,
   }
 
   override def invalidSettings: Seq[(String,String)] = {
+    if (name.trim.isEmpty) {
+      return Seq("Variable" -> I18N.gui.get("edit.behaviorSpace.name.empty"))
+    }
+    for (exp <- experimentNames) {
```

Comment:
I'd probably just do `if (experimentNames.contains(name.trim)) ...` here for clarity.